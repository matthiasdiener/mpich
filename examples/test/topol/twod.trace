Note: Line numbers for system and library calls may be incorrect 
Begin traceback...
Called from [func: (null)], at 0xef702cf4, args=0xa 0x2 0xef7d1a7c 0xef802960
Called from [func: _MPI_Recv], at 0x26f68, args=0x7ad44 0x7ad44 0xf6010 0xf6010
Called from [func: _MPIR_Op_setup], at 0x19dd8, args=0xf6010 0x7ad44 0xeffff344 0xeffff348
Called from [func: _MPI_Reduce], at 0x2d7f4, args=0x7ad30 0x7ad44 0x1 0xf23f0
Called from [func: _MPIR_Op_setup], at 0x1a014, args=0x7ad30 0x7ad44 0x1 0xf23f0
Called from [func: _MPI_Allreduce], at 0xff2c, args=0x7ad30 0x7ad44 0x1 0xf23f0
Called from [func: _mpi_allreduce_], at 0x430c, args=0x7ad30 0x7ad44 0x1 0xf23f0
Called from [func: _MAIN_], at 0x2660, args=0x7ad30 0x7ad44 0x745c8 0x78924
Called from [func: (null)], at 0xef77cc38, args=0x0 0x1 0x1 0x2e000000
Called from [func: start], at 0x2064, args=0x0 0x10 0xeffff664 0x74400
End traceback...
