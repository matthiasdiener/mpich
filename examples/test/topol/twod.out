*** Testing twod from Fortran ***
*** Testing twod from Fortran ***
